online-store/webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const { merge } = require('webpack-merge');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const EslintPlugin = require('eslint-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const baseConfig = {
+    entry: path.resolve(__dirname, './src/index'),
+    mode: 'development',
+    module: {
+        rules: [
+            {
+                test: /\.css$/i,
+                use: ['style-loader', 'css-loader'],
+            },
+            {
+                test: /\.ts$/i,
+                use: 'ts-loader'
+            },
+            {
+                test: /\.(?:ico|gif|png|jpg|jpeg|svg)$/i,
+                type: 'asset/resource',
+                generator: {
+                    filename: 'img/[name][ext]'
```

Comment:
I would suggest to include [hash] in it. Currently if you change something in your images client will keep getting old images because of browser cache (it happens because name of the image after compiling has not been changed). I know it might cause some additional issues with image srcs. You don't have to jump into it but it is important so I are informed now